src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -339,9 +336,9 @@ public function makePathRelative($endPath, $startPath)
         $endPathRemainder = implode('/', array_slice($endPathArr, $index));
 
         // Construct $endPath from traversing to the common path, then to the remaining $endPath
-        $relativePath = $traverser.(strlen($endPathRemainder) > 0 ? $endPathRemainder.'/' : '');
+        $relativePath = $traverser.($endPathRemainder !== '' ? $endPathRemainder.'/' : '');
```

Comment:
We are using Yoda condition in Symfony, so it should be `'' !== $endPathRemainder` instead.